### PR TITLE
added goerli and sepolia inverse colors

### LINKED ISSUE #16787: Add inverse network colors to design system consts
### Description
Add `GOERLI_INVERSE` and `SEPOLIA_INVERSE` to `COLORS` in `ui/helpers/constants/design-system.js`. Check that those options work with the `Box` component 

### Technical Details
- Add `GOERLI_INVERSE` and `SEPOLIA_INVERSE` to `COLORS` in `ui/helpers/constants/design-system.js`
- Ensure above mappings use correct design token css variables

### Acceptance criteria 
- `COLORS.GOERLI_INVERSE` works as an option for the `Box` component
- - `COLORS.SEPOLIA_INVERSE` works as an option for the `Box` component

### PR DESCRIPTION
**Pull request by NidhiKJha**
This PR is to update `COLORS.SEPOLIA_INVERSE` and `COLORS.GOERLI_INVERSE `

Fixes: #16787

### CODE REVIEW

File: ui/components/ui/box/box.stories.js [hunk 1]
```diff
@@ -346,6 +346,12 @@ export const Color = (args) => {
       <Box {...args} padding={3} color={COLORS.INHERIT} {...args}>
         COLORS.INHERIT
       </Box>
+      <Box {...args} padding={3} color={COLORS.SEPOLIA_INVERSE} {...args}>
+        COLORS.SEPOLIA_INVERSE
+      </Box>
+      <Box {...args} padding={3} color={COLORS.GOERLI_INVERSE} {...args}>
+        COLORS.GOERLI_INVERSE
+      </Box>
```
Comment (georgewrmarshall) [inline ui/components/ui/box/box.stories.js:354]: Can you add the network defaults as background colors as well?

```suggestion
      <Box {...args} padding={3} backgroundColor={COLORS.SEPOLIA} color={COLORS.SEPOLIA_INVERSE} {...args}>
        COLORS.SEPOLIA_INVERSE
      </Box>
      <Box {...args} padding={3} backgroundColor={COLORS.GOERLI} color={COLORS.GOERLI_INVERSE} {...args}>
        COLORS.GOERLI_INVERSE
      </Box>
```

**Review by georgewrmarshall — COMMENTED** [review]
Nice! Can you also add it to the TEXT_COLORS and ICON_COLORS consts respectively they should have the same coverage as PRIMARY_DEFAULT/PRIMARY_INVERSE

Could add to the controls for color
Screenshot 2022-12-13 at 9 47 20 AM

Could you also add GOERLI and SEPOLIA to the Background Color story 

Screenshot 2022-12-13 at 9 42 17 AM

**Review by darkwing — DISMISSED** [review]

File: ui/components/ui/box/box.stories.js [hunk 2]
```diff
@@ -346,6 +346,24 @@ export const Color = (args) => {
...
+        {...args}
+      >
+        COLORS.SEPOLIA_INVERSE
+      </Box>
+      <Box
+        {...args}
+        padding={3}
+        backgroundColor={COLORS.GOERLI}
+        color={COLORS.GOERLI_INVERSE}
+        {...args}
```
Comment (georgewrmarshall) [inline ui/components/ui/box/box.stories.js:363]: Ahh looks like I was spreading `{...args}` twice here 🤦 could you please remove the duplication for the entire story/file 🙏

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Hmm just realized that the inverse colors for sepolia and goerli should be white. Doesn't look like the class name is being attached and even when I added `box--text-color-sepolia-inverse` it didn't seem to work so not sure if the class name is being generated either?

I think you also need to add the inverse options to `ui/css/design-system/colors.scss`

Screenshot 2022-12-14 at 9 09 28 AM

**Comment by NidhiKJha** [comment]
> Hmm just realized that the inverse colors for sepolia and goerli should be white. Doesn't look like the class name is being attached and even when I added `box--text-color-sepolia-inverse` it didn't seem to work so not sure if the class name is being generated either?
> 
> I think you also need to add the inverse options to `ui/css/design-system/colors.scss`
> 
> Screenshot 2022-12-14 at 9 09 28 AM

Nice Catch! Updated it here 

Screenshot 2022-12-14 at 10 54 44 PM

**Review by georgewrmarshall — APPROVED** [review]
Nicely done 🤙 💯

**Review by garrettbear — APPROVED** [review]

**Pull request merged**